package.json

```diff
@@ -1,10 +1,57 @@
 {
+  "license": "Apache-2.0",
   "private": true,
   "workspaces": [
     "@shared/api",
     "@shared/constants",
     "docs",
     "extension",
     "@stellar/lyra-api"
-  ]
+  ],
+  "scripts": {
+    "build": "run-p --print-label build:lyra-api build:docs build:extension",
+    "build:netlify": "run-s build:lyra-api build:docs",
+    "build:lyra-api": "yarn workspace @stellar/lyra-api build",
+    "build:docs": "yarn workspace docs build",
+    "build:extension": "yarn workspace extension build",
+    "start": "run-p --print-label start:lyra-api start:docs start:extension start:unpacked",
+    "start:lyra-api": "yarn workspace @stellar/lyra-api start",
+    "start:docs": "yarn workspace docs start",
```

Comment:
so I think we still have the problem of if I clone this repo fresh, and I just want to `start` docs, running this command will fail unless I've already built lyra-api

Reply:
I did a test clone, and it appears to finish building fast enough that it doesn't break. I definitely think it's a race condition that happens to work out, though

Reply:
if you just run `yarn start:docs`? shouldn't you need to do: `run-s build:lyra-api yarn workspace docs start` similar to the netlify build?

Reply:
It's working when I try it from scratch 🤷 
```sh
$ git clone ~/workspace/lyra/
$ cd lyra/
$ yarn
$ yarn start
```
goes off without a hitch locally

```

Reply:
Ah but you said specifically `yarn start:docs`. That does break, but I thought the happy path was good enough 🤷 I think it's relatively infrequent we'd only start docs

Reply:
yeah, it may be infrequent, but I think the start commands should work "out of the box" if we're including them

Reply:
oh i see - bc `yarn start` aliases these, you don't want to build every time

Reply:
Right exactly. Could work around that by having `docs:start` run the `@stellar/lyra-api` build first with like, `docs:start:quick` that skips that or something, but since it's infrequent and because `"scripts"` is getting pretty busy, I thought I'd avoid yet another script